src/azure/devops/workflow/pre-defined-tasks/pre-defined-task-json-patch-document-mapper.ts

```diff
@@ -0,0 +1,71 @@
+import { JsonPatchDocument, Operation } from "azure-devops-node-api/interfaces/common/VSSInterfaces";
+import { Mapper } from "../../../../mapper";
+import { PreDefinedTask } from "./pre-defined-task";
+
+export class PreDefinedTaskJsonPatchDocumentMapper
+    implements Mapper<PreDefinedTask, JsonPatchDocument> {
+
+    private static readonly TitleFieldPath: string = '/fields/System.Title';
+    private static readonly DescriptionFieldPath: string = '/fields/System.Description';
+    private static readonly RemainingWorkFieldPath: string = '/fields/Microsoft.VSTS.Scheduling.RemainingWork';
+    private static readonly AreaPathFieldPath: string = '/fields/System.AreaPath';
+    private static readonly IterationPathFieldPath: string = '/fields/System.IterationPath';
+    private static readonly RelationsFieldPath: string = '/relations/-';
+    private static readonly AssignedToFieldPath: string = '/fields/System.AssignedTo';
+
+    constructor(
```

Comment:
The fields injected with this constructor aren't needed if we abstract them away.